File: langgraph/data/semantic_equiv/27.py
def invert_character_case(input_text: str) -> str:
    result = ''
    for char in input_text:
        if char.islower():
            result += char.upper()
        elif char.isupper():
            result += char.lower()
        else:
            result += char
    return result
# method name: flip_case -> invert_character_case
# parameter name: string -> input_text
# replaced swapcase() with manual character case inversion using a loop and conditionals

File: langgraph/result/qwen/14b-eq2js/27.js
function invertCharacterCase(inputText) {
    let result = '';
    for (let char of inputText) {
        if (char === char.toLowerCase()) {
            result += char.toUpperCase();
        } else if (char === char.toUpperCase()) {
            result += char.toLowerCase();
        } else {
            result += char;
        }
    }
    return result;
}

// method name: flip_case -> invertCharacterCase
// parameter name: string -> inputText
// replaced swapcase() with manual character case inversion using a loop and conditionals